lib/tasks/add_health_questions.rake

```diff
@@ -6,24 +6,20 @@ desc <<-DESC
   Add health questions to a vaccine.
 
   Usage:
-    rake add_health_questions[team_id,vaccine_id,replace]
+    rake add_health_questions[programme,vaccine_id,replace]
 
   The vaccine must belong to the team given, this is a safety check.
 
   Use "replace" for the replace arg to replace the existing health questions.
 
   Example:
 
-    rake add_health_questions[1,1,replace]
+    rake add_health_questions[hpv,1,replace]
 DESC
 task :add_health_questions,
-     %i[team_id vaccine_id replace] => :environment do |_task, args|
-  team = Team.find(args[:team_id])
-  vaccine =
-    team
-      .programmes
-      .flat_map(&:vaccines)
-      .find { |v| v.id == args[:vaccine_id].to_i }
+     %i[programme vaccine_id replace] => :environment do |_task, args|
+  programme = Programme.find_by!(type: args[:programme])
+  vaccine = programme.vaccines.find_by(id: args[:vaccine_id])
```

Comment:
I think this breaks as there are references to `team` further down, and the `raise` below needs to be updated too as we don't pass in `team` anymore

Reply:
Yep, I pushed a little too quickly there, should be working now.